pkg/services/user/userimpl/user.go

```diff
@@ -254,6 +254,21 @@ func (s *Service) GetSignedInUser(ctx context.Context, query *user.GetSignedInUs
 	return signedInUser, err
 }
 
+func (s *Service) NewAnonymousSignedInUser(ctx context.Context) (*user.SignedInUser, error) {
```

Comment:
not 100% sure about this, but should we return an error if 

https://github.com/grafana/grafana/blob/94ed7444541a5cb4ce8ee13e81aa0207a3ed3631/pkg/setting/setting.go#L371

is false?

Reply:
makes sense to me. added with https://github.com/grafana/grafana/pull/57537/commits/812d146acd38362c01a0a3ca31379ba3d242061c

Reply:
Nice, lgtmm!